### DISCUSSION TITLE
Unraid Community App

### DISCUSSION THREAD

**Discussion by LilTrublMakr** [cross-seed/cross-seed#302] [General]
Just wanted to reach out and let @mmgoodnow know that I made a template that is set up for the Community App store so end users don't have to create the app manually. I kept it close to the guide but made some (what I think are) minor improvements. If you would like anything changed, please let me know.

-LTM

**Comment by mmgoodnow** [comment]
Thank you very much! Do you have a link that I can check out? Also, I'd love it if you could update the docs because I'm very unfamiliar with Unraid. https://github.com/cross-seed/cross-seed.org/edit/master/docs/tutorials/unraid.md

**Comment by LilTrublMakr** [comment]
There isn't rally a good way for you to see the full functionality of it because Unraid servers are self hosted... [This](https://unraid.net/community/apps?q=cross-seed) is about the best I can do. 

It basically just takes that tutorial and adds an 'install' button.

As for the doc, I did a very rough update. Feel free to clean it up as you wish: https://gist.github.com/LilTrublMakr/a1836cfa52ff1cd3a8398fd0ace23517

**Reply by zakkarry** [reply]
> There isn't rally a good way for you to see the full functionality of it because Unraid servers are self hosted... [This](https://unraid.net/community/apps?q=cross-seed) is about the best I can do.
> 
> It basically just takes that tutorial and adds an 'install' button.
> 
> As for the doc, I did a very rough update. Feel free to clean it up as you wish: https://gist.github.com/LilTrublMakr/a1836cfa52ff1cd3a8398fd0ace23517

We've had our own official container I've been maintaining for quite some time now, and I was curious if you would have any objection to removing yours to prevent the confusion that has taken place (yours being older I assume means it shows up first), especially given the likelihood that additional changes will be made once v6 is published as stable.

![image](https://github.com/user-attachments/assets/3844df10-6028-4309-be60-ba984aab76cd)

**Reply by LilTrublMakr** [reply]
Definitely. I don't use UNRAID anymore, is there a tag I can add to the template to archive it or deprecate it so it hides it from the list?

**Reply by zakkarry** [reply]
> Definitely. I don't use UNRAID anymore, is there a tag I can add to the template to archive it or deprecate it so it hides it from the list?

whaaaaa!!!! traitor :P what did you switch to TrueNAS or proxmox or something?

Let me double check on this, but I think you might be able to just delete the XML.

**Reply by zakkarry** [reply]
The responses I'm getting are to use the forums and/or contact Squid (community apps guy) who you probably spoke with to set it up in the first place - he's who I spoke with.

Let me know if this is possible still, I assume you have an account active there still.

**Reply by LilTrublMakr** [reply]
Both actually! Gave me really good practice for my current job with HA, permissions, and stuff like that. Plus I got rid of my power hog of a server and went with Micro PCs.

**Reply by zakkarry** [reply]
If it doesn't pull over 150w with all the drives spun up its not a real server :P

**Reply by zakkarry** [reply]
Let me know if you still have an active account on unraid's forums, if not I can make a post and link to this thread and your repository to substantiate what I'd be saying.

**Reply by LilTrublMakr** [reply]
Was pulling around 165w without the drives spinning... Was a Dell R720XD.

Messaged Squid asking to archive my apps.

**Reply by zakkarry** [reply]
Oh, that's a bit much for spun down. Idle mine pulls around 130w at the UPS (includes router/AP) - under load its 160-170w. Consumer hardware though, only 7 spinners in my array + 2 nvme. I never spin anything down.

Power is cheap where I live though.

**Reply by LilTrublMakr** [reply]
Well I sent a message to Squid and he doesn't appear to have seen it yet...

**Reply by zakkarry** [reply]
When i checked he hadn't been on since Saturday or something, I'm sure he'll see it.

You could always try renaming the XML file and see if it just disappears if you dont want to wait. I really just care about what happens when we release v6 - if it's archived or removed by then.

Keep me posted please.

**Reply by LilTrublMakr** [reply]
Still nothing. I deleted the XML. Lets see if it clears.

**Reply by zakkarry** [reply]
I asked if he was on vaction or something in the discord, they didn't seem to know of anything that would keep him from answering.

If the XML deletion is not working, they recommended I just ping him on Discord.

We'll see what happens, thanks for the efforts :)

**Reply by zakkarry** [reply]
Not sure if you heard back from him or deleting the XML worked, but it appears to be gone now.

Appreciate you working with us on this!

![image](https://github.com/user-attachments/assets/ef069654-cd20-4a22-93cb-4fcbe3861030)

**Reply by LilTrublMakr** [reply]
Must have been deleting the XML. Squid still has not seen my PM. Glad all is good now... Now about integration for secret managers... Lol
